content/yaml/building-a-native-android-app.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Building a native Android app
+description: Building an Android app with YAML.
+weight: 3
+---
+## Building an Android app
+
+The necessary command for building native Android application goes under `scripts` in the [overall architecture](../yaml/yaml/#template) in the `codemagic.yaml` file. For Android (built with gradle), the script looks like this:
+
+    - ./gradlew bundleDebug
+
+## Testing an Android app
+
+The code for testing an Android app also goes under `scripts`. A few examples of testing can be found [here](../yaml/testing).
+
+## Code signing
```

Comment:
It shouldn't be a separate subsection since it would be an empty one. I'd rather add the references for code signing and publishing information either as bullet points or in notebox.